src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-details/rgw-bucket-details.component.html

```diff
@@ -28,54 +28,14 @@
       </tr>
       <tr>
         <td i18n
-            class="bold">Marker</td>
-        <td>{{ selection.marker }}</td>
-      </tr>
-      <tr>
-        <td i18n
-            class="bold">Maximum marker</td>
-        <td>{{ selection.max_marker }}</td>
-      </tr>
-      <tr>
-        <td i18n
-            class="bold">Version</td>
-        <td>{{ selection.ver }}</td>
-      </tr>
-      <tr>
-        <td i18n
-            class="bold">Master version</td>
-        <td>{{ selection.master_ver }}</td>
-      </tr>
-      <tr>
-        <td i18n
-            class="bold">Modification time</td>
+            class="bold">Last modification time</td>
         <td>{{ selection.mtime | cdDate }}</td>
       </tr>
-      <tr>
-        <td i18n
-            class="bold">Zonegroup</td>
-        <td>{{ selection.zonegroup }}</td>
-      </tr>
-      <tr>
-        <td i18n
-            class="bold">Versioning</td>
-        <td>{{ selection.versioning }}</td>
-      </tr>
-      <tr>
-        <td i18n
-            class="bold">Encryption</td>
-        <td>{{ selection.encryption }}</td>
-      </tr>
-      <tr>
-        <td i18n
-            class="bold">MFA Delete</td>
-        <td>{{ selection.mfa_delete }}</td>
-      </tr>
     </tbody>
   </table>
 
   <!-- Bucket quota -->
-  <div *ngIf="selection.bucket_quota">
+  <div>
```

Comment:
But shouldn't we show quota only if quota is enabled for given bucket https://docs.ceph.com/en/latest/radosgw/admin/#enable-disable-bucket-quota

Reply:
That's what I did, earlier it was showing even if the quota was not enabled.

![Screenshot from 2023-08-09 17-41-15](https://github.com/ceph/ceph/assets/6695156/a24653ac-39ac-4680-b030-e11ba7245431)

![image](https://github.com/ceph/ceph/assets/6695156/9f797afd-98ab-4947-bae0-21ad0accb33c)